src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
You're linking a SCSS file in your HTML. Browsers can't interpret SCSS, they can only interpret CSS. You should compile your SCSS to CSS and then link the CSS file in your HTML.